packages/fx-core/src/component/generator/generatorProvider.ts

```diff
@@ -1,21 +1,25 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { SpecGenerator } from "./apiSpec/generator";
 import { CopilotExtensionGenerator } from "./copilotExtension/generator";
 import { DefaultTemplateGenerator } from "./defaultGenerator";
 import { OfficeAddinGeneratorNew } from "./officeAddin/generator";
+import { CustomEngineAgentWithExistingApiSpecGenerator } from "./openApiSpec/customEngineAgentGenerator";
+import { DeclarativeAgentWithExistingApiSpecGenerator } from "./openApiSpec/declarativeAgentGenerator";
+import { MessageExtensionWithExistingApiSpecGenerator } from "./openApiSpec/messageExtensionGenerator";
 import { SsrTabGenerator } from "./other/ssrTabGenerator";
 import { TdpGenerator } from "./other/tdpGenerator";
 import { SPFxGeneratorImport, SPFxGeneratorNew } from "./spfx/spfxGenerator";
 
 // When multiple generators are activated, only the top one will be executed.
 export const Generators = [
   new TdpGenerator(),
-  new OfficeAddinGeneratorNew(),
-  new SsrTabGenerator(),
   new DefaultTemplateGenerator(),
+  new OfficeAddinGeneratorNew(),
   new SPFxGeneratorNew(),
   new SPFxGeneratorImport(),
-  new SpecGenerator(),
+  new SsrTabGenerator(),
```

Comment:
What's the purpose to adjust the order? I think it may break some .dotnet template

Reply:
If DefaultGenerator does not conflict with SSRTabGenerator, then it's ok.